docs/GettingStartedDocs/OP-TEE/Hardware/ScalysTrustBox.md

```diff
@@ -0,0 +1,393 @@
+I/TC: Successfully captured Cyres certificate chain
+I/TC: Successfully captured Cyres private key
+I/TC: Initialized
+Trying to boot from RAM
+```
+
+If HAB is enabled, the messages will instead look as follows:
+
+```
+U-Boot SPL 2018.09-00480-gdc28a9fa63-dirty (Jan 17 2019 - 11:17:15 -0800)
+PPA Firmware: Version LSDK-18.09
+SEC Firmware: 'loadables' present in config
+loadables: 'trustedOS@1'
+I/TC:
+I/TC: OP-TEE version: v0.4.0-443-g9cdcf55b-dev #6 Sat Jan 26 05:59:52 UTC 2019 aarch64
+I/TC: Successfully captured Cyres certificate chain
+I/TC: Successfully captured Cyres private key
+I/TC: Initialized
+Trying to boot from RAM
+```
+
+### Recovery
+
+If upon reset the board fails to boot, you can repeat the procedure to re-enter
+recovery U-Boot to flash the firmware again. Flashing the firmware does not
+overwrite copy of U-Boot, which is part of the recovery ROM.
+
+## Root File System
+
+In this subsection, you will build a root filesystem comprised of Linux, Ubuntu
+18.04 user-mode as well as the necessary libraries and supporting binaries to
+communicate with OP-TEE OS. You will then flash the resulting filesystem onto
+a MicroSD card.
+
+### Building the Filesystem
+
+In the same folder where you cloned the LSDK repository, do:
+
+```bash
+make os
+```
+
+**Note:** Do not attempt to use `-j` here either. The steps that are
+parallelizable will be parallelized automatically.
+
+### Flashing the Filesystem
+
+Plug in the MicroSD card into your system, determine which block device node
+corresponds to the MicroSD card, then issue the following command from within
+the directory where you cloned the LSDK repository:
+
+```bash
+make sdcard DEV=/dev/sdX
+```
+
+When the script finishes, it is safe to remove the MicroSD card.
+
+**Note:** You might observe an error related to copying an `Image` file. This is
+expected if the MicroSD card is empty; copying this file is part of a backup
+step. `make` is configured to ignore the error and continue.
+
+### Booting the Filesystem
+
+Insert the MicroSD card you just flashed into the TrustBox and power it up.
+In the `minicom` window you should see Linux booting up.
+
+The default login credentials are:
+
+```
+Username: root
+Password: root
+```
+
+To log in over SSH, issue the `ifconfig` command to see your board's IP address,
+if it is connected to a network.
+
+## OP-TEE OS Test Suite
+
+To ensure that the build of OP-TEE OS is sane, first start the TEE supplicant on
+the board:
+
+```bash
+tee-supplicant &
+```
+
+You need only do this once per boot. Then, issue:
+
+```
+xtest
+```
+
+**Note:** `xtest` causes failures on purpose; do not be alarmed by the numerous
+stack traces scrolling by.
+
+Once `xtest` finishes, you should see the following output:
+
+```
+24078 subtests of which 1 failed
+95 test cases of which 1 failed
+0 test cases were skipped
+TEE test application done!
+```
+
+The single failed test is a known issue.
```

Comment:
is there a GitHub issue filed to track this?  if so, can you insert the issue number here so in case it's fixed and this page is missed at that time, the reader can verify the status?

Reply:
Good idea. Created an issue and added a link.